src/stream/binary_reader.rs

```diff
@@ -418,7 +418,7 @@ mod tests {
     fn streaming_parser() {
         use crate::stream::Event::*;
 
-        let reader = File::open(&Path::new("./tests/data/binary.plist")).unwrap();
+        let reader = File::open(Path::new("./tests/data/binary.plist")).unwrap();
```

Comment:
yep, there's a bunch of those, but those pre-existed. I just removed the referencing because clippy complained, but you're right of course, we don't need to create a Path here. I'll remove them.

Reply:
it's been done ^^